<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="refreshButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="controlpanel.qrc">
          <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="DEChooserButton">
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="controlpanel.qrc">
          <normaloff>:/images/kde.png</normaloff>:/images/kde.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="DELaunchConfigApp">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="controlpanel.qrc">
          <normaloff>:/images/kde.png</normaloff>:/images/kde.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="filterEdit">
        <property name="placeholderText">
         <string>What do you want to configure?</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>603</width>
         <height>484</height>
        </rect>
       </property>
       <widget class="QCheckBox" name="softwareGroupName">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>6</y>
          <width>471</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox{ background-color: rgb(160, 160, 160)}
QCheckBox::indicator:unchecked{
image: url(:images/arrow-down.png);
}
QCheckBox::indicator:checked{
image: url(:images/arrow-up.png);
}</string>
        </property>
        <property name="text">
         <string>Software and updates</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="softwareGroupCheckbox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>173</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox{ background-color: rgb(160, 160, 160)}
QCheckBox::indicator:unchecked{
image: url(:icons/arrow-down.png);
}
QCheckBox::indicator:checked{
image: url(:icons/arrow-up.png);
}</string>
        </property>
        <property name="text">
         <string>System</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QAutoExpandList" name="softwareLW">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>28</y>
          <width>501</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QAutoExpandList" name="listWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>501</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QAutoExpandList</class>
   <extends>QListWidget</extends>
   <header location="global">controls/qautoexpandlist.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="controlpanel.qrc"/>
 </resources>
 <connections/>
</ui>
